docs/ir/ir-breaking-changes.rst

```diff
@@ -138,7 +138,7 @@ The function ``preincr_u8(1)`` returns the following values:
 
 .. index:: ! evaluation order; function arguments
 
-On the other hand, function argument expressions are evaluated in the same order by both code generators.
+On the other hand, function argument expressions are evaluated in the same order by both code generators with two exceptions.
```

Comment:
Maybe just say "with the exception of the global functions `addmod` and `mulmod`".

Also as it reads now, you might think that the example is for addmod / mulmod.

Reply:
Updated.